validator_client/src/block_service.rs

```diff
@@ -269,7 +253,7 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
                                     .await;
                             },
                             Err(BlockError::Irrecoverable(e))  => {
-                                error!(log, "Error whilst producing a blinded block, cannot fallback because block was signed"; "error" => ?e);
+                                error!(log, "Error whilst producing a blinded block, cannot fallback because the block was signed"; "error" => ?e);
```

Comment:
Could wrap this

Reply:
Added here: 4c67d7d625274c8c9eb9effc7f0405b3f5095c92